charcoal/just_taxonomy.py

```diff
@@ -340,7 +340,7 @@ def main(args):
     print(f'loaded {len(tax_assign)} tax assignments.')
 
     with open(args.matches_sig, 'rt') as fp:
-        siglist = list(sourmash.load_signatures(fp))
+        siglist = list(sourmash.load_file_as_signatures(fp))
```

Comment:
I _think_ this has to be `load_file_as_signatures(args.matches_sig)`, with the open/etc. `load_file_as_signatures` does not take a file handle.

Reply:
good catch thank you!

Reply:
(let's note this down as something that needs to be covered by tests! it shouldn't work as is!)